File: DSApython/Algorithm/Sorting/selectionSort.py
def selectionSort(arr):
    n = len(arr)
    for i in range(n-1):
        for j in range(i+1,n):
            if arr[i] > arr[j]: arr[i],arr[j] = arr[j],arr[i]

arr = [9,8,7,6,5,4,3,2,1]
selectionSort(arr)
print(arr)

File: DSAcpp/Algorithm/Sortings/selectionSort.cpp
#include <iostream>
using namespace std;

void selectionSort(int arr[], int n){
    for(int i=0;i<n-1;i++){
        for(int j=i+1;j<n;j++)
            if(arr[i]>arr[j]) swap(arr[i],arr[j]);
    }
}

int main(){
    int arr[] = {10,9,8,7,6,5,4,3,2,1};
    int n = sizeof(arr)/sizeof(arr[0]);
    selectionSort(arr,n);
    for(int i=0;i<n;i++) cout << arr[i] << ' ';
    cout << '\n';
}